docker_utils/rtenvsub.sh

```diff
@@ -0,0 +1,386 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 SD Elements Inc.
+#
+#  All Rights Reserved.
+#
+# NOTICE:  All information contained herein is, and remains
+# the property of SD Elements Incorporated and its suppliers,
+# if any.  The intellectual and technical concepts contained
+# herein are proprietary to SD Elements Incorporated
+# and its suppliers and may be covered by U.S., Canadian and other Patents,
+# patents in process, and are protected by trade secret or copyright law.
+# Dissemination of this information or reproduction of this material
+# is strictly forbidden unless prior written permission is obtained
+# from SD Elements Inc..
+# Version
+
+version='0.1'
+
+# Set a safe umask
+umask 0077
+
+# Log/Console destination
+console="$(tty || logger || false)"
+
+# Import the standard shell library
+# shellcheck source=../shtdlib.sh disable=SC1091
+source "$(dirname "${0}")/../shtdlib.sh" &> /dev/null || ../shtdlib.sh &> /dev/null || source ./shtdlib.sh &> /dev/null || source shtdlib.sh
+
+
+debug 10 "Running ${0} with PID: ${$}"
+
+if ! whichs envsubst ; then
+    color_echo red "Unable to locate envsubst command, please make sure it's available"
+    color_echo cyan 'Perhaps this can be fixed with: apt-get -y install gettext-base'
```

Comment:
Just a nitpick, but is there a reason why you switch between double quotes and single quotes for lines 34/35 and 39/40?

Reply:
I should be using single quotes everywhere since no variable substitution is needed, thx

Reply:
Err it's because the first sentence has a ' in it so I used double quotes for that one

Reply:
Ah I somehow missed that, thanks.